bidadjustment/build_rules_test.go

```diff
@@ -446,11 +513,80 @@ func TestMerge(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "NotNilExtPrebid-NotNilExtBidAdj-NotNilAcct",
+			givenRequestWrapper: &openrtb_ext.RequestWrapper{
+				BidRequest: &openrtb2.BidRequest{Ext: []byte(`{"prebid":{"bidadjustments":{"mediatype":{"banner":{"bidderA":{"dealId":[{ "adjtype": "multiplier", "value": 1.1}]}}}}}}`)},
+			},
+			acctBidAdjustments: &openrtb_ext.ExtRequestPrebidBidAdjustments{
+				MediaType: openrtb_ext.MediaType{
+					VideoInstream: map[openrtb_ext.BidderName]openrtb_ext.AdjustmentsByDealID{
+						"bidderB": {
+							"dealId": []openrtb_ext.Adjustment{{Type: AdjustmentTypeCPM, Value: 3}},
+						},
+					},
+					VideoOutstream: map[openrtb_ext.BidderName]openrtb_ext.AdjustmentsByDealID{
```

Comment:
Nitpick: Just to make the test case less verbose, can we remove some of the elements so we are only left with bidder `"bidderA"` and `"bidderB"`?

Reply:
it's necessary to test all media types `banner|video-instream|video-outstream|audio|native|*`